edsnlp/pipelines/misc/measurements/patterns.py

```diff
@@ -1,4 +1,11 @@
 number_terms = {
+    "0.125": ["⅛"],
+    "0.16666666": ["⅙"],
+    "0.2": ["⅕"],
+    "0.25": ["¼"],
+    "0.3333333": ["⅓"],
+    "0.5": ["½"],
+    "2.5": ["21/2"],
```

Comment:
```suggestion
    "2.5": ["5/2"],
```

Reply:
Ah indeed, it does collides with dates but the component does not match over existing entities or dates. And 21/2 is most more often a wrongly parsed fraction than a date. I feel like the risk for false positive is low, since it has to be followed by a unit like 21/2 cm.

Reply:
thanks